tests/common/fixtures/ptfhost_utils.py

```diff
@@ -231,7 +231,7 @@ def setup_vlan_arp_responder(ptfhost, rand_selected_dut, tbinfo):
     logger.info("Start arp_responder")
     ptfhost.command('supervisorctl start arp_responder')
 
-    yield vlan, ipv4_base, ipv6_base
+    yield vlan, ipv4_base, ipv6_base, ip_offset
```

Comment:
Since you changed the signature of function `setup_vlan_arp_responder`, please do a global search to ensure all the references are updated.

Reply:
@bingwang-ms the fixture "setup_vlan_arp_responder" is only used in the tests/arp/test_arp_update.py currently.